### DISCUSSION TITLE
Github Account Taken Down Without any Notice or any mail.

### DISCUSSION THREAD

**Discussion by HVGaruda** [community/community#103178] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My Main Github Account is taked down by github without any notice aur any email no-one is able to view my github account rather than me is there any way to get my account back.

**Comment by gil-son** [comment]
You must to contact [Github Support](https://support.github.com/) 

I hope I was helpfull. Good luck!

**Comment by daguerpedro** [comment]
It happened to me, the support said:
> Sometimes our abuse detecting systems highlight accounts that need to be manually reviewed. 
> We've cleared the restrictions from your account, so you have full access to GitHub again. 
> Please let us know if we can help with anything else.  
  
But few days before my account were flagged again and no warnings were sent, it happend again on January 7 and then January 31.
The solution is simple: open a ticket at https://support.github.com/ and talk to then, every time you notice your account is down, reopen the ticket. (Or delete that account and create another like i did)

**Reply by HVGaruda** [reply]
If this type of thing happens again and again what's the sense of using Github we dont't have so much time to waste it with github even after knowing this that they will not respond support i think now it's better to use other platform.

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @CoderPrit! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, the best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by HVGaruda** [comment]
Account released.
